packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -301,13 +319,46 @@ const createFlowsheetResultCellDisplay = (resultDataSet, hideUnit, numericOverfl
         }
       }
     });
+
     const displayCount = additionalResultCount + 1;
-    const hasAccessoryIcons = (firstResultAttributes.comment || firstResultAttributes.modified || firstResultAttributes.unverified);
-    const additionalResultsStackDisplay = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
+    const additionalResultsStack = createEndAdditionalResultsStack(displayCount, additionalResultInterpretations, hasAccessoryIcons, resultKeyID);
```

Comment:
Can we add a new line here too? Between the const declaration and the comment.

Reply:
Yup! https://github.com/cerner/terra-clinical/blob/1feea097b7fad242ae9e7d622d5ec465c11c3688/packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx#L329